package.json

```diff
@@ -19,38 +20,43 @@
   "license": "MPL2",
   "dependencies": {
     "alameda": "^1.0.0",
+    "fxos-mvc": "fxos-eng/mvc#master",
```

Comment:
Yeah, that's indented for npm modules that are distributed, but we have an app that's not distributed, hence the confusion :)